style.css

```diff
@@ -5,9 +5,11 @@ body {
 
 .card {
     background: #ffffff;
+    border-radius: 10px;
+    -moz-box-shadow: 3px 3px 14px 0px rgba(27,43,63,0.64);
+    box-shadow: 3px 3px 14px 0px rgba(27,43,63,0.64);
```

Comment:
Great thinking with adding shadow, I wanted to this but didn't have the time. these shadows area bit harsh, lets reduce the blur down a bit. a light shadow will do the trick. a blur at about 5px is all that is needed. Also reduce border-radius to 5px, so we have a slight curve and not hard circle like edges.